content/swiftui/concepts/color/color.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'Color'
+Description: 'Creates a color instance.'
```

Comment:
Since this is a concept, it doesn't need to abide by the terms layout. See: https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md

Reply:
Can we describe it as a structure? In addition, we can go into the different color models, e.g., RGB, Grayscale.

There are also standard colors (type properties, which I think would do well as term entries under this concept) which can be passed into certain modifiers.